src/zgossip.c

```diff
@@ -114,6 +114,11 @@ struct _server_t {
 
     tuple_t *cur_tuple;         //  Holds current tuple to publish
     zgossip_msg_t *message;     //  Message to broadcast
+
+#ifdef CZMQ_BUILD_DRAFT_API
```

Comment:
this structure, and the static function as well, are internal only so for simplicity there's no need to wrap them in ifdefs

Reply:
ack